examples/window/window_settings.rs

```diff
@@ -32,16 +32,16 @@ fn main() {
 
 /// This system toggles the vsync mode when pressing the button V.
 /// You'll see fps increase displayed in the console.
-fn toggle_vsync(input: Res<Input<KeyCode>>, mut windows: ResMut<Windows>) {
+fn toggle_vsync(input: Res<Input<KeyCode>>, mut windows: Query<&mut Window>) {
```

Comment:
Another bug: while the scale factor override now works, the system scale factor is ignored. If I change the scale factor on my machine from 1 to 2, the physical resolution remains the same (500x300) when it should double to (1000x600). The scale factor also remains at 1, which feels like the core issue.

Reply:
I just fixed this!